src/common/components/Item/index.spec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import proxyquire from 'proxyquire';
+import { stubComponent, stubStyles } from 'test/utils';
+
+const TooltipTrigger = stubComponent('TooltipTrigger');
+const ResourceLink = stubComponent('ResourceLink');
+const Icon = stubComponent('Icon');
+const Gw2Icon = stubComponent('Gw2Icon');
+
+const Item = proxyquire.noCallThru().noPreserveCache()('common/components/Item', {
+  'common/components/TooltipTrigger': TooltipTrigger,
+  'common/components/ResourceLink': {
+    buildLink: function(a,b) { return ''; },
+  },
+  'common/components/Icon': Icon,
+  'common/components/Gw2Icon': Gw2Icon,
+});
```

Comment:
bit of a tricky thing with proxyquire but you need to access `.default` here, e.g:

```
const Item = proxyquire.noCallThru().noPreserveCache()('common/components/Item', {
  'common/components/TooltipTrigger': TooltipTrigger,
  'common/components/ResourceLink': ResourceLink,
  'common/components/Icon': Icon,
  'common/components/Gw2Icon': Gw2Icon,
}).default;
```

also the resource link is fine, change it back to what i have above :)

Reply:
Man, this is frustrating. That worked, but now I cant seem to get the TooltipTrigger, and then ultimately get its attributes. I'd like to assert that the TooltipTrigger gets the correct type, either 'skins' or 'items'.

I've tried all of the enzyme selectors http://airbnb.io/enzyme/docs/api/selector.html with no success.

It's like its not rendering the TooltipTrigger, but there is nothing that should cause it to return before rendering logic.

Reply:
Ok, I think I got it. I dont know what 'recompose/pure' is doing, but it was causing the issue. I stubbed it out to just return the exported Item class and now the test passes.

I'll add more tests tomorrow and fix-up the tooltip text from your other comment.

Reply:
ah yeah you're correct. `recompose/pure` is a higher order component that stops rendering from happening if the props were the same. :)

good job so far man, sounds fantastic